### DISCUSSION TITLE
Docs: Migration from vite shows app router docs

### DISCUSSION THREAD

**Discussion by gavinthomas-valtech** [vercel/next.js#68912] [Help]
### What is the update you wish to see?

I would like to see specific steps to migrate from vite to pages router. pages router doesn't have `app` directory

### Is there any context that might help us understand?

The migration page shows the same guidance for both app and pages router. pages router doesn't have client components

### Does the docs page already exist? Please link to it.

https://nextjs.org/docs/pages/building-your-application/upgrading/from-vite

**Comment by samcx** [comment]
@gavinthomas-valtech Thank you for submitting an issue!

Any Documentation "requests" should use the other template instead (go to our [Discussions](https://github.com/vercel/next.js/discussions).

![CleanShot 2024-08-14 at 13 58 44@2x](https://github.com/user-attachments/assets/e6e05c86-9dac-4600-b360-d30e8fd3eef3)

Yeah, we don't have a direct Vite to App Router guide. But you could migrate from Pages to App Router afterwards, or just use this guide (Vite and CRA should be similar since they're classic SPAs) → https://nextjs.org/docs/app/building-your-application/upgrading/from-create-react-app.
